package-schema-registry/include/etc/confluent/docker/ensure

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset \
+    -o errexit \
+    -o verbose \
+    -o xtrace
+if [[ -n "${SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL-}" ]]
+then
+    echo "===> Check if Zookeeper is healthy ..."
+    cub zk-ready "$SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL" "${SCHEMA_REGISTRY_CUB_ZK_TIMEOUT:-40}"
+fi
+
+echo "===> Check if Kafka is healthy ..."
+
+if [[ -n "${SCHEMA_REGISTRY_KAFKASTORE_SECURITY_PROTOCOL-}" ]] && [[ $SCHEMA_REGISTRY_KAFKASTORE_SECURITY_PROTOCOL = "SSL" ]]
```

Comment:
This condition should be !PlAINTEXT

https://github.com/confluentinc/cp-docker-images/blob/d73bbde30bff9e9ea2f543f8be6e24b4a66e8e21/debian/schema-registry/include/etc/confluent/docker/ensure#L29

Reply:
Yeah, I will bring in the latest changes from cp-docker-images. There was a little drift since opening this PR.